projects/observability/src/shared/services/csv-downloader/csv-downloader.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+import { Dictionary } from '@hypertrace/common';
+import { CsvDownloadFileConfig, FileDownloadService } from '@hypertrace/components';
+import { Observable, of } from 'rxjs';
+
+@Injectable({ providedIn: 'root' })
+export class CSVDownloaderService {
```

Comment:
nit: `GlobalCsvDownloadService` would be a better name I guess
We also need to put comments on how and when to use this

Reply:
done